libavcodec/vvc/vvc_thread.c

```diff
@@ -266,6 +267,11 @@ static int run_parse(VVCContext *s, VVCLocalContext *lc, VVCTask *t)
             if (next < sc->eps + sc->nb_eps) {
                 memcpy(next->cabac_state, ep->cabac_state, sizeof(next->cabac_state));
                 av_assert0(!next->parse_task->type);
+                for (size_t i = 0; i < FF_ARRAY_ELEMS(lc->ep->stat_coeff); ++i) {
```

Comment:
better define a function ep_init_stat_coeff, call it here and in cabac_init_state